examples/02-custom-vpc-cidr-no-nodes.yaml

```diff
@@ -12,5 +12,9 @@ metadata:
 vpc:
   cidr: 10.10.0.0/16
   autoAllocateIPv6: true
+  # by default clusters are publicly exposed and disabled for private access
```

Comment:
the comment here and values are not in sync I think

Reply:
right... I struggled writing this comment ...
I'm trying to explain the default and here we show how we can override the default.
Let me improve that